### ISSUE TITLE
[bug] no UI error when fail during kustomize build

### ISSUE THREAD

**Issue by kwladyka** [argoproj/argo-cd#1651]
**Describe the bug**

v1.0.0 argo cd

There is no UI info about a bug. It looks like on screen endless.

![image](https://user-images.githubusercontent.com/3903726/58432747-3e590200-80b4-11e9-92a2-d8cda73aca17.png)

`kubectl -n argocd log pod/argocd-repo-server-5cddbdc75c-bqfh4`

```
time="2019-05-25T08:22:05Z" level=error msg="`kustomize build /tmp/git@gitlab.com:cemspred/gitops/argo-cd-clusters-bootstraping/production` failed: Error: AccumulateTarget: evalsymlink failure on '/tmp/git@gitlab.com:etingroup_gitops-k8s/gitops/argo-cd-clusters-bootstraping/base/transformer-config.yaml' : lstat /tmp/git@gitlab.com:censored/gitops/argo-cd-clusters-bootstraping/base/transformer-config.yaml: no such file or directory\n\n"
```

I missed 1 file for kustomization.yaml
```
configurations:
- transformer-config.yaml
```

**To Reproduce**
```
configurations:
- transformer-config.yaml
```

in kustomization.yaml but do not create this file. I guess it should have the same effect.

**Expected behavior**
UI error

I would like to see output logs in such cases. It is not a first time when UI doesn't predict something and there is no way to debug what really happen.

**Comment by alexec** [comment]
Was there anything under "details" please? Otherwise, I think this would be a good first issue.

**Comment by kwladyka** [comment]
It wasn't.

**Comment by alexec** [comment]
Fixed in v1.2
